pages/Messages/Overview.md

```diff
@@ -38,8 +38,8 @@ Some announcements, such as broadcasts and replies, will contain references to c
 
 #### Content
 
-Content items, or simply content, refers to an activity pub compliant object hosted at some URI and intended to be posted via an announcement in a batch.
-Generally, content within the DSNP specification will be defined by the [W3C Activity Pub specification](https://www.w3.org/TR/activitypub/#object-without-create), however extensions, such as the [Mastodon Activity Pub specification](https://docs.joinmastodon.org/spec/activitypub/) may also be implemented.
+Content items, or simply content, refers to activity streams compliant JSON hosted at some URI and intended to be posted via an announcement in a batch.
```

Comment:
Not related to this PR, but I'm wondering if we should be referring to URLs instead of URIs since there is an expectation that they represent locations where content can be retrieved.

Reply:
URI vs URL is a debate that continues to rage on :) Let's discuss for a future PR.